about.css

```diff
@@ -0,0 +1,1065 @@
+    margin-top: 40px;
+    display: grid;
+    grid-template-columns: 50% 50%;
+  }
+
+  .sec3 .speakers .guest {
+    display: flex;
+    -moz-column-gap: 16px;
+    column-gap: 16px;
+    height: 100%;
+  }
+
+  .sec3 .speakers .guest img {
+    position: relative;
+    background-size: 120px 100px;
+    background-repeat: no-repeat;
+    width: 150px;
+    height: 150px;
+    object-fit: cover;
+    padding-left: 6%;
+    padding-top: 6%;
+  }
+
+  .about .about-sec1 .logo img {
+    width: 25rem;
+    height: 20rem;
+  }
+
+  .sec3 .speakers .guest img:hover {
+    transform: translate(1px, -1px);
+    box-shadow: 0 3px 10px var(--color-black);
+  }
+
+  .sec3 .speakers .guest section {
+    display: flex;
+    flex-direction: column;
+    gap: 14px;
+    height: 100%;
+  }
+
+  footer .last-abt h3 {
+    color: var(--color-white);
+    display: flex;
+  }
+
+  footer .foot-last h3 {
+    font-weight: lighter;
+    display: flex;
+    flex-direction: column;
+    width: 40%;
+    justify-content: center;
+    align-items: center;
+    font-size: 14px;
+    padding: 30px 0;
+  }
+
+  .sec3 .speakers .guest section h3 {
+    align-self: flex-start;
+  }
+
+  .sec3 .speakers .guest section .p1 {
+    width: 100%;
+    font-size: 15px;
+    color: var(--color-secondaryDeep);
+  }
+
+  .sec3 .speakers .guest section .p2 {
+    font-size: 13px;
+    color: var(--color-lightblack);
+    width: 100%;
+  }
+
+  .about .about-sec1 {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+  }
+
+  .sub-about {
+    background-image: url(./important-img/aboutdesktop-bg.png);
+    background-position: 0 0;
+    background-repeat: no-repeat;
+    box-sizing: border-box;
+    background-size: contain;
+    display: flex;
+    padding: 35px 12% 130px;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+  }
+
+  .about .about-sec1 p a {
+    display: block;
+  }
+
+  .about .about-sec1 p a b {
+    color: var(--color-black);
+  }
+
+  .about .about-sec1 .about-title {
+    color: var(--color-lightblack);
+    font-size: 50px;
+    padding: 8rem 0 3rem 0;
+  }
+
+  .about .about-sec1 .logo {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    width: 100%;
+    margin: 1rem 0 6.5rem 0;
+  }
+
+  .about .about-sec1 .mini-about {
+    color: var(--color-lightblack);
+    font-size: 50px;
+    margin-top: 20px;
+  }
+
+  .about .about-sec1 .images-about {
+    padding-top: 20px;
+    margin: 4rem;
+    display: flex;
+    flex-direction: row;
+    width: 90%;
+    text-align: center;
+    gap: 24px;
+  }
+
+  .about .about-sec1 .images-about .img-about {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    width: 90%;
+    align-self: center;
+    background-color: rgba(226, 34, 12, 0.6);
+  }
```

Comment:
- [ ] This section should contain a section header, a divider, a line of text, and two images with text centered over them. Also, the image has an opacity, in yours is difficult to see the text that is over the text. Kindly fix that 🙃 😉

Your design

<img width="954" alt="Screenshot 2023-03-24 at 20 37 55" src="https://user-images.githubusercontent.com/74501523/227603820-618b9b5d-b6a4-4b76-905b-b7f744feb153.png">

Original design
 
<img width="855" alt="Screenshot 2023-03-24 at 20 40 00" src="https://user-images.githubusercontent.com/74501523/227603921-a07d6f8b-2ed8-424f-920c-e7f9fd6ca278.png">